cmd/internal/ecs/aws_api.go

```diff
@@ -87,8 +87,19 @@ func wrapUnauthorizedFor(err error, arn arn) error {
 // Check whether we can load the config object
 func (wf *AddWorkflow) createConfig() error {
 	// Try loading credentials for the profile.
-	// TODO: specify a location inside the Docker container, if running on Docker
-	sharedConfig, err := config.LoadSharedConfigProfile(wf.ctx, wf.awsProfile)
+	// Use the default if running standalone.  Add /aws for use inside a container.
+	configFiles := []string{config.DefaultSharedConfigFilename(), "/aws/config"}
+	credentialsFiles := []string{config.DefaultSharedCredentialsFilename(), "/aws/credentials"}
+	if awsCredentialsFlag != "" {
+		credentialsFiles = append(credentialsFiles, awsCredentialsFlag)
+	}
```

Comment:
Does the order here matter? We should make sure the user-specified files take precedence over the default files.

Reply:
I believe that's what it does, but the documentation complicates things:

> 	// If duplicate profiles are provided with a same, or across multiple
>	// shared credentials files, the next parsed profile will override only
>	// properties that conflict with the previously defined profile. Note that
>	// if duplicate profiles are provided within the SharedCredentialsFiles and
>	// SharedConfigFiles, the properties defined in shared credentials file
>	// take precedence.

I'm reading a lot into "next" but I believe that means it visits them in order and applies changes as it goes.

Reply:
Gotcha, thanks. Your interpretation looks right.